includes/Admin/Settings.php

```diff
@@ -30,6 +30,7 @@ public function __construct() {
         add_filter( 'dokan_admin_localize_script', [ $this, 'add_admin_settings_nonce' ] );
         add_action( 'wp_ajax_dokan_refresh_admin_settings_field_options', [ $this, 'refresh_admin_settings_field_options' ] );
         add_filter( 'dokan_get_settings_values', [ $this, 'format_price_values' ], 12, 2 );
+        add_filter( 'dokan_settings_general_site_options', [ $this, 'add_dokan_data_clear_setting' ], 310 );
```

Comment:
you don't need to add filter, kindly add this setting directly into existing settings array

Reply:
@nurul-umbhiya bhai
Unable to place this setting at end of `General>>Site Options` without filter.

Reply:
file: includes/Admin/Settings.php
under function get_settings_fields()

Reply:
as we have access to our codebase, we don't need to use hooks unless necessary